### ISSUE TITLE
Pagination url is not forming correctly (Broken)

### ISSUE THREAD

**Issue by rajjanorkar** [ARCANEDEV/LogViewer#198]
- LogViewer Version: 4.3
- Laravel Version: 5.4
- PHP Version: 7

### Description:
Pagination not working properly 

`http://localhost.test/log-viewer/logs`

when i click on second page it is redirecting to the 

`http://localhost.test/?page=2`

Not retaining actual url

### Steps To Reproduce:

1. Go to log-viewer dashboard 
2. Click on Logs (make sure you have many logs so you can see pages)
3. Click on any page at the top or bottom or click on next/previous
4. You will notice it has broken the pagination it will redirect to home page.

**Comment by arcanedev-maroc** [comment]
Sorry but i can't reproduce it.

I've just tested the LogViewer `v4.3.9` with Laravel `v5.4.36`, it works fine for me.

Try to debug this [method](https://github.com/ARCANEDEV/LogViewer/blob/master/src/Entities/LogEntryCollection.php#L45-L57) and check the `$path` variable is not null in your side.

**Comment by rajjanorkar** [comment]
It is definitely not working with default setup all my other paginations work perfectly fine.

**Comment by arcanedev-maroc** [comment]
Try to debug this method: https://github.com/ARCANEDEV/LogViewer/blob/master/src/Entities/LogEntryCollection.php#L45-L57

Do something like this:

```php
public function paginate($perPage = 20)
{
    $page      = request()->get('page', 1);
    $path      = request()->url();
    $paginator = new LengthAwarePaginator(
        $this->forPage($page, $perPage),
        $this->count(),
        $perPage,
        $page,
        compact('path')
    );

    dd($path, $paginator); // SEND ME THE OUTPUT OF THIS

    return $paginator;
}
```

**Comment by arcanedev-maroc** [comment]
It's working even with `artisan serve`:

![2018-03-03_19-22-10](https://user-images.githubusercontent.com/3282340/36938385-b7c5613c-1f18-11e8-818b-415129dc4894.jpg)

**Comment by rajjanorkar** [comment]
Pagination on your attached page works fine. it is not working when you are on 
`http://localhost:8000/log-viewer/logs`

**Comment by rajjanorkar** [comment]
The above method you mentioned is for paginating per day, which works fine. But the problem is on logs for days list.

```
"//localhost:8000/log-viewer/logs/2018-03-04"

LengthAwarePaginator {#316 ▼
  #total: 147
  #lastPage: 5
  #items: LogEntryCollection {#23805 ▶}
  #perPage: 30
  #currentPage: 1
  #path: "//localhost:8000/log-viewer/logs/2018-03-04"
  #query: []
  #fragment: null
  #pageName: "page"
}
```

**Comment by arcanedev-maroc** [comment]
Patch released: v4.3.10

Hope this helps you to solve your issue :+1:

**Comment by rajjanorkar** [comment]
Thank you @arcanedev-maroc that fixed the pagination issue.
